resolwe/flow/executors/__init__.py

```diff
@@ -56,8 +56,8 @@ def iterjson(text):
 def hydrate_spawned_files(filename, data_id):
     """Hydrate spawned files' paths."""
     if filename not in EXPORTED_FILES_MAPPER:
-        raise KeyError('All files referenced in spawned processes must be exported using'
-                       '`re-export` command.')
+        raise KeyError('All files referenced in spawned processes must be exported using '
```

Comment:
`re-export` is not used in spawned process, but for spawned process, so I modified the message a bit.